tests/test_hawkular.py

```diff
@@ -27,19 +29,20 @@ def provider():
     A stub urlopen() implementation that load json responses from
     the filesystem.
     """
+    if not os.getenv('HAWKULAR_HOSTNAME'):
```

Comment:
we are mocking rest api calls, but in __init__ method of Hawkular it calls rest api, so we need to mock it before instance creation